components/admin_console/billing/company_info.tsx

```diff
@@ -1,17 +1,28 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
-import {FormattedMessage} from 'react-intl';
+import React, {useEffect} from 'react';
+import {useDispatch, useStore} from 'react-redux';
+
+import {DispatchFunc} from 'mattermost-redux/types/actions';
+import {getCloudCustomer} from 'mattermost-redux/actions/cloud';
 
 import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
-import BlockableLink from 'components/admin_console/blockable_link';
+
+import CompanyInfoDisplay from './company_info_display';
 
 type Props = {
 
 };
 
 const CompanyInfo: React.FC<Props> = () => {
+    const dispatch = useDispatch<DispatchFunc>();
+    const store = useStore();
+
+    useEffect(() => {
+        getCloudCustomer()(dispatch, store.getState());
```

Comment:
Better to dispatch this action as opposed manually invoking the func.

```suggestion
        dispatch(getCloudCustomer());
```